app/templates/components/settings/contact-info-section.hbs

```diff
@@ -1,11 +1,11 @@
 <form class="ui form" {{action 'submit' on = 'submit'}}>
   <div class="field">
     <label>Email</label>
-    <input type="text" name="email" placeholder="Email">
+    {{input type="text" name="email" placeholder="Email"}}
```

Comment:
@niranjan94 In case of value parameter, don't you think it would make user erase the value inside the input and then type what he want? Can't only placeholder do the work? It looks much cleaner. Opinions?